src/Modules/Workspaces/Commands.Workspaces/GetPowerBIWorkspace.cs

```diff
@@ -27,7 +27,7 @@ public class GetPowerBIWorkspace : PowerBIGetCmdlet, IUserScope, IUserFilter, IU
         private const string ListParameterSetName = "List";
 
         // Since internally, users are null rather than an empty list on workspaces v1 (groups), we don't need to filter on type for the time being
-        private const string OrphanedFilterString = "(not users/any()) or (not users/any(u: u/groupUserAccessRight eq Microsoft.PowerBI.ServiceContracts.Api.GroupUserAccessRight'Admin'))";
+        private string OrphanedFilterString = string.Format("(state ne '{0}') and ((not users/any()) or (not users/any(u: u/groupUserAccessRight eq Microsoft.PowerBI.ServiceContracts.Api.GroupUserAccessRight'Admin')))", WorkspaceState.Deleted);
```

Comment:
Consider using string interpolation. Did you have this change reviewed with PMs and checked against the backend to verify that it returns the correct results?

Reply:
Yes, this change was discussed with the PM and also tested , verified against Edog.